frontend/src/components/Navbar.js

```diff
@@ -44,37 +50,37 @@ function NavbarFun() {
               </Nav>
             </>
           )}
-          <Navbar.Toggle aria-controls="navbarScroll" />
+          <Navbar.Toggle aria-controls="navbarScroll" onClick={() => setExpanded(!expanded)}/>
```

Comment:
Use of the previously defined handleLinkClink on the navbar on the navbar components. Looking good.